BookingProject/BookingWebAPI/BookingWebApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BookingWebApi.Application.DTOs.AppUserDTOs;
+using BookingWebApi.Application.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookingWebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly IAuthenticationService _authService;
+
+        public AccountController(IAuthenticationService authService)
+        {
+            _authService = authService;
+        }
+
+        [HttpPost("Register")]
+        public async Task<IActionResult> Register(RegisterDto registerDto)
+        {
+            var result = await _authService.Register(registerDto);  
+
+            if(result.Success)
+            {
+                return Ok(result.Value);
+            }
+            else
+            {
+                return BadRequest(result);
+            }
```

Comment:
You might create extension method to wrap this logic. Then you will have `result.ToResponse()`.
You might omit Unauthorized status code and just return BadRequest for failed login, it's ok practice